README.md

```diff
@@ -73,6 +73,7 @@ with the following options:
 - List of extensions: `*.twig`
 - Invocation:
     - Parser command: `<project>/vendor/bin/twig-gettext-extractor --sort-output --force-po -o %o %C %K -L PHP --files %F`
+        - (Windows user having problems? Try to use this instead `C:\\Users\path\to\your\project\vendor\bin\twig-gettext-extractor.bat --sort-output --force-po -o %o %C %K -L PHP --files %F`)
```

Comment:
Mainly yes. Without the `.bat` extension it doesn't work. I don't know if Poedit should resolve hte `<project>` tag depending on the path in which I am saving the .po file or not. I guess it does not. Therefore I think it is better to write a fake path, in this way users of this fantastic project can start using it faster without trying multiple times the Poedit configuration.